[2025-03-30T02:36:35.154+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-30T02:36:35.169+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T02:36:32.865649+00:00 [queued]>
[2025-03-30T02:36:35.177+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T02:36:32.865649+00:00 [queued]>
[2025-03-30T02:36:35.178+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-30T02:36:35.192+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-03-30 02:36:32.865649+00:00
[2025-03-30T02:36:35.201+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4657) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-30T02:36:35.204+0000] {standard_task_runner.py:72} INFO - Started process 4659 to run task
[2025-03-30T02:36:35.203+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_data_pipeline', 'fetch_data', 'manual__2025-03-30T02:36:32.865649+00:00', '--job-id', '561', '--raw', '--subdir', 'DAGS_FOLDER/Gestioncommands.py', '--cfg-path', '/tmp/tmp7nznj7e6']
[2025-03-30T02:36:35.205+0000] {standard_task_runner.py:105} INFO - Job 561: Subtask fetch_data
[2025-03-30T02:36:35.253+0000] {task_command.py:467} INFO - Running <TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T02:36:32.865649+00:00 [running]> on host 778b271a0247
[2025-03-30T02:36:35.338+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='ecommerce_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T02:36:32.865649+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T02:36:32.865649+00:00'
[2025-03-30T02:36:35.340+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-30T02:36:35.341+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-30T02:36:35.342+0000] {logging_mixin.py:190} INFO - Current task name:fetch_data state:running start_date:2025-03-30 02:36:35.170587+00:00
[2025-03-30T02:36:35.343+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline and current dag run status:running
[2025-03-30T02:36:35.343+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-30T02:36:35.345+0000] {Gestioncommands.py:37} INFO - Fetching data from API...
[2025-03-30T02:36:35.967+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host 'my.api.mockaroo.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-03-30T02:36:36.720+0000] {Gestioncommands.py:41} INFO - Data fetched successfully
[2025-03-30T02:36:36.722+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-30T02:36:36.730+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-30T02:36:36.731+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ecommerce_data_pipeline, task_id=fetch_data, run_id=manual__2025-03-30T02:36:32.865649+00:00, execution_date=20250330T023632, start_date=20250330T023635, end_date=20250330T023636
[2025-03-30T02:36:36.749+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-30T02:36:36.750+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-30T02:36:36.750+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline queued_at:2025-03-30 02:36:32.875768+00:00
[2025-03-30T02:36:36.751+0000] {logging_mixin.py:190} INFO - Task hostname:778b271a0247 operator:PythonOperator
[2025-03-30T02:36:36.784+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-30T02:36:36.806+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T02:36:36.808+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
